src/sentry/services/hybrid_cloud/user/service.py

```diff
@@ -193,7 +192,6 @@ def get_user(self, user_id: int) -> RpcUser | None:
 
         :param user_id: The user to fetch
         """
-        metrics.incr("user_service.get_user.call")
```

Comment:
I don't need these metrics anymore. They have proven the theory I had on cache hit rates.